content/en/docs/contributing/localization.md

```diff
@@ -11,90 +11,89 @@ localizations. English is the default language, with US English as the default
+### Summary / TL;DR {#summary}
 
-<!-- markdownlint-disable blanks-around-fences -->
+Do translate:
 
-- Do **not** fix the broken links. Each non-English page is associated with a
-  specific commit of the corresponding English page, as identified by the git
-  commit hash value of the `default_lang_commit` front matter key.
-- Configure the link checker to ignore the non-English pages by adding the
-  following to the page's front matter, or to the closest common ancestor file,
-  when more than one page has link errors:
-  ```yaml
-  htmltest:
-    # TODO: remove the IgnoreDirs once broken links are fixed
-    IgnoreDirs:
-      - path-regex/to/non-en/directory/contain/files/to/ignore
-      - path-2-etc
-  ```
-- Run `npm run check:links` and include any updates to the `.htmltest.yml`
-  config file with your PR.
+- [Front matter][] fields `title`, `linkTitle`, and `description`
+- Page content, including the text fields in Mermaid [diagrams](#images)
 
-<!-- markdownlint-enable blanks-around-fences -->
+Do **NOT**:
 
-## Translation guidance
+- Translate:
+  - **File or directory** names of resources in this repository
+  - [Links](#links), this includes [heading IDs](#headings).[^*]
+  - [Front matter][] fields other than those listed in the "Do" section above.
+    In particular, do not translate `aliases`. When in doubt, ask maintainers.
+- Create **copies of images**, unless you [localize text in the images](#images)
 
-We recommend that you follow the guidance offered in this section when
-translating pages.
+[^*]: For a possible exception, see [Links](#links).
 
-### Heading anchors
+### Heading IDs {#headings}
 
 To ensure that heading anchor targets are uniform across localizations, when
 translating headings:
 
-- Preserve the heading's explicit ID if it has one. [Heading ID syntax] is
+- Preserve the heading's explicit ID if it has one. [Heading ID syntax][] is
   written after the heading text using syntax like `{ #some-id }`.
 - Otherwise, explicitly declare a heading ID corresponding to the autogenerated
   ID of the original English heading.
 
-{{% alert title="Note" %}}
-
-We leave it to the discretion of localization authors to decide if they add an
-explicit heading ID to all headings of translated pages, or whether this is only
-done for known intra-site heading targets, as reported by the link checker. The
-former option is more uniform and more work. It better supports having
-site-external targets into localization pages and avoids having to revisit
-previously translated pages as new heading targets are used.
-
-{{% /alert %}}
-
 [Heading ID syntax]:
   https://github.com/yuin/goldmark/blob/master/README.md#headings
 
-### Links
+### Links {#links}
 
-For link references to local paths (as opposed to external links), preserve the
-path _as is_. This holds true for links to website pages or section-local
-resources such as images.
+Do not translate link references, keep them unchanged. This holds true for links
```

Comment:
```suggestion
Do not translate link references. Keep them unchanged. This holds true for links
```

Reply:
I'd rather keep those phrases together. IMHO, "Keep them unchanged" doesn't make sense as a standalone sentence. I can, if you prefer, just drop the "keep them unchanged".

Reply:
Since "keep them unchanged" is a bit redundant, I'm in favor of dropping it.